modules/swagger-codegen/src/test/java/io/swagger/codegen/CodegenTest.java

```diff
@@ -482,10 +482,10 @@ public void fromModelObjectTest() throws NoSuchFieldException, IllegalAccessExce
     @DataProvider(name = "fromModelEnumTest")
     public static Object[][] fromModelEnumTest() {
         return new Object[][]{
-                {"IntegerEnum", true, false, "integer", Arrays.asList(1, 2, 3)},
-                {"LongEnum", false, false, "long", Arrays.asList(1, 2, 3) /* this seems wrong, the SwaggerParser should read the enum values as Long not as Integer */},
-                {"FloatEnum", false, true, "float", Arrays.asList("1", "2", "3") /* this seems wrong, the SwaggerParser should read the enum values as Float not as String */},
-                {"DoubleEnum", false, false, "double", Arrays.asList("1", "2", "3") /* this seems wrong, the SwaggerParser should read the enum values as Double not as String */},
+                {"IntegerEnum", true, false, "integer", Arrays.asList("1", "2", "3") /* @see: https://github.com/swagger-api/swagger-core/issues/2449 */},
```

Comment:
If this test fails it is probably because the mentioned bug was solved and the swagger-core was updated in the swagger-codegen.

This link shall help to understand that there was a bug which was solved and because of that the test is now failing and can be corrected.

Without that link the contributor can only guess how the tests might be.


But I can remove the link and write a big fat comment explaining the whole situation and that test might fail in the future and what to do if that happens. I thought a link might shorter and is more capable of explaining the test situation.